data/test/scenarios/test_return.cfg

```diff
@@ -15,3 +15,37 @@
         {RETURN ([not][true][/true][/not])}
     [/event]
 )}
+
+# Same as the above two, but using the Lua unit_test module
```

Comment:
It's good that there are now four tests, but the documentation no longer matches.